Formula/raylib.rb

```diff
@@ -1,10 +1,10 @@
 class Raylib < Formula
   desc "Simple and easy-to-use library to learn videogames programming"
   homepage "http://www.raylib.com/"
-  url "https://github.com/raysan5/raylib/archive/1.9.1-dev.tar.gz"
-  version "1.9.1-dev"
-  sha256 "892357fb44d340eb7449c23c425d660d98b34b91434400e7610514ef02698600"
-  head "https://github.com/raysan5/raylib.git", :branch => "develop"
+  url "https://github.com/raysan5/raylib/archive/1.9.4-dev.tar.gz"
```

Comment:
The latest version is marked as “Pre-release” on their github

Reply:
Can I have brew install --dev install the last tagged release instead and reset this to 1.9.1?

Reply:
Ah didn't notice the build failure, I've removed the version bump. Still I would like to have --dev install the newest tag if possible